spec/policy_machine_storage_adapters/active_record_spec.rb

```diff
@@ -107,6 +107,207 @@
           expect(user.linked?(pm2_user)).to eq true
         end
       end
+
+      describe '#bulk_persist' do
+        let(:pm) { PolicyMachine.new(name: 'AR PM', storage_adapter: PolicyMachineStorageAdapter::ActiveRecord) }
+        let(:user) { pm.create_user('alice') }
+        let(:caffeinated) { pm.create_user_attribute('caffeinated') }
+        let(:decaffeinated) { pm.create_user_attribute('decaffeinated') }
+
+        describe 'policy element behavior' do
+          it 'deletes a policy element that has been created and then deleted in a persistence buffer' do
```

Comment:
Given that all these specs end with `in a persistence buffer`, it might be worth just putting them in a context block.